tests/test_ec2/test_route_tables.py

```diff
@@ -618,3 +618,39 @@ def test_describe_route_tables_with_nat_gateway():
     nat_gw_routes[0]["DestinationCidrBlock"].should.equal("0.0.0.0/0")
     nat_gw_routes[0]["NatGatewayId"].should.equal(nat_gw_id)
     nat_gw_routes[0]["State"].should.equal("active")
+
```

Comment:
Can you add some test cases for the different VPC types ('Interface'|'Gateway'|None).

Also add some asserts that either DNS entries are created, or routes - but not both.